### DISCUSSION TITLE
Change the design of the detected document overlay

### DISCUSSION THREAD

**Discussion by mvanroon** [WebsiteBeaver/react-native-document-scanner-plugin#76] [Ideas]
First of all, thank you very much for this fantastic library. The code looks very good. We'll probably be implementing this library in our fintech app to allow our users to scan passports, drivers licenses and debit cards.

However, our designer is not very pleased with the aesthetics of the current overlay of the detected document (semi-transparent squares) on iOS.

I'm not sure of this is a VisionKit default that cannot be changed, but ideally we'd either like to be able to:

 - Configure the styles of the overlays of detected documents
 - Turn off the overlay completely and have a callback when a document is detected or not detected so we could add our own indicator to show if the camera is currently detecting a document.

Let me know if this is something that is feasible. I might be able to get my client to sponsor this feature if that helps.

**Comment by dmarcs** [comment]
Hi, unfortunately VisionKit doesn't allow this type of customization.

**Reply by mvanroon** [reply]
Thanks for the reply. Do you know of any alternatives out there?
